src/Core/Caching/ScopedCacheClient.cs

```diff
@@ -1,13 +1,18 @@
-﻿using System;
+﻿using Foundatio.Extensions;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace Foundatio.Caching {
     public class ScopedCacheClient : ICacheClient {
+        private readonly string _keyPrefix;
+
         public ScopedCacheClient(ICacheClient client, string scope) {
             UnscopedCache = client ?? new NullCacheClient();
             Scope = scope;
+
+            _keyPrefix = scope != null ? String.Concat(scope, ":") : String.Empty;
```

Comment:
I don't think the consumer needs to know about the `:`? Would be strange to pass a scope into the ctor and have the `Scope` property return something (slightly) different.